packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -486,4 +492,29 @@ router.get(
   })
 );
 
+router.post(
+  "/trigger-update",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const patients = await getPatients({ cxId });
+    const chunks = chunk(patients, patientChunkSize);
+
+    for (const chunk of chunks) {
+      await Promise.allSettled(
```

Comment:
NABD, but I suggest using `executeAsynchronously` so we avoid rewriting this kind of code over and over - and minimize the risk of introducing bugs.

Reply:
ill pass changing the code here if that's ok, but ill keep it in mind for the future

Reply:
I was going to add that I think we should avoid adding things that are not pushing towards the patterns we have/direction we want to go, and the change is very small, but you already merged the PR.